ElectronicObserver/Window/Control/Paging/PagingControlViewModel.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+
+namespace ElectronicObserver.Window.Control.Paging;
+
+public partial class PagingControlViewModel : ObservableObject
+{
+	public int CurrentPage { get; set; } = 0;
+
+	private ObservableCollection<object?> Items { get; set; } = new();
+
+	public List<object?> ItemsPaged { get; set; } = new();
```

Comment:
`set` can be private.

Reply:
Maybe `DisplayedItems` would be a better name?

Reply:
`private set` & name changed to `DisplayedItems `